<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMappingComponent</class>
 <widget class="QTabWidget" name="CMappingComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tb_CurrentMappings">
   <attribute name="title">
    <string>Current mappings</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_CurrentMappings">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="BlackGui::Views::CAircraftModelView" name="tvp_CurrentMappings">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_AircraftModels">
   <attribute name="title">
    <string>Aircraft models</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_AircraftModels">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="BlackGui::Views::CAircraftModelView" name="tvp_AircraftModels">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Views::CAircraftModelView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/aircraftmodelview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
